recipes/protobuf/all/conanfile.py

```diff
@@ -299,6 +299,8 @@ def package_info(self):
                 self.cpp_info.components["libprotobuf-lite"].system_libs.append("log")
             if self._protobuf_release >= "22.0":
                 self.cpp_info.components["libprotobuf-lite"].requires.extend(absl_deps)
+                if not self.options.shared:
+                    self.cpp_info.components["libprotobuf-lite"].requires.extend(["utf8_validity"])
```

Comment:
✅  this comes from https://github.com/protocolbuffers/protobuf/blob/8476dbacb35dfb4b149256581467603cb005708c/cmake/libprotobuf-lite.cmake#L49